azure-quantum/azure/quantum/qiskit/results/resource_estimator.py

```diff
@@ -6,7 +6,63 @@
 To install run: pip install azure-quantum[qiskit]"
     )
 
+import markdown
+
 class ResourceEstimatorResult(Result):
+    def __init__(
+        self,
+        **kwargs
+    ) -> None:
+        super().__init__(**kwargs)
+        self.summary = ResourceEstimatorResultSummary(**kwargs)
+
+    def _repr_html_(self):
+        return self._repr_html_success_() if self.success else self._repr_html_error_()
+
+    def _repr_html_success_(self):
+        html = ""
+
+        md = markdown.Markdown(extensions=['mdx_math'])
+        for group in self.data()['reportData']['groups']:
+            html += f"""
+                <details {"open" if group['alwaysVisible'] else ""}>
+                    <summary style="display:list-item">
+                        <strong>{group['title']}</strong>
+                    </summary>
+                <table>"""
```

Comment:
(Nit) For me the `<table>` indentation in line 32 (L32) is shown the same as the indentation for `<details` in L28. This makes an impression of a mismatch between the opening and closing tag.
Consider increasing the indentation for `<table>` (to the same as for `</summary>` in L31).

Reply:
Fixed this.